### ISSUE TITLE
Keras model

### ISSUE THREAD

**Issue by jfdelgad** [Google-Health/path-foundation#8]
the model provided in hugging face is a  SavedModel and this creates all kind of issued, in particular if the idea is to finetune it.

using tf.keras this works as expected, the training accuracy increases as well as the validation when doing transfer learning but it is not possibly to set the variables to trainable. using tf_keras, the results are sometimes good some times bad, I repeated this 10 times with 40 epochs on a small datset (after 40 epochs the accuracy is basically random in 2 out of 5 repetitions with the same data). using tf_keras is also extremely slow (minutes using tf.keras vs hours using tf_keras) in a A100 GPU even if the path_foundational model parameters trainable field is set to False

can we get a .keras model for this?
